fix #2030 about nginx (#155)
Varying-Vagrant-Vagrants/varyingvagrantvagrants.org: docs/en-US/adding-a-new-site/custom-public_html.md

The default site provisioners will generate a `public_html` sub folder that acts as the sites webroot. This is different from custom paths and folders, as move the entire site, including error logs, provisioners, etc.

> If you want to change which folder gets served to the browser, this is what you want. you might do this because the server you're replicating uses `htdocs` instead of `public_html`.

Reviewer: ```suggestion
You can also change the public webroot to another folder. You might do this because the server you're replicating uses `htdocs` instead of `public_html`.
```

## How It Gets Set

> This is set in the Nginx config file in `provision/vvv-nginx.conf`, check the [dedicated page](https://varyingvagrantvagrants.org/docs/en-US/adding-a-new-site/custom-nginx/).

Reviewer: This page doesn't actually say what to change :/ I think instead, we should keep the original text, and add 2 sentences

> This is set in the Nginx config file in `provision/vvv-nginx.conf`, check the [dedicated page](https://varyingvagrantvagrants.org/docs/en-US/adding-a-new-site/custom-nginx/).

Reviewer: ```suggestion
This is set in the Nginx config file. Specifically, on a line that looks like this:

\```Nginx
\  root {vvv_path_to_site}/public_html;
\```

`{vvv_path_to_site}` gets replaced with the location of your site on the filesystem, and to change to `htdocs`, simply replace `public_html` with `htdocs`, save, then reprovision. Likewise, you can go deeper down if your git repository has a more complex structure.

[See here for more information on custom Nginx config files](https://varyingvagrantvagrants.org/docs/en-US/adding-a-new-site/custom-nginx/).
```

It seems you can't put code blocks inside code suggestions :/ I added `\` so it would show the whole suggestion
